README.md

```diff
@@ -520,9 +509,9 @@ _Why:_
     _Why:_
     > This is a very well-known design to developers (your main API consumers). Apart from readability and ease of use, it allows us to write generic libraries and connectors without even knowing what the API is about.
 
-* use kebab-case for URLs.
+* use camelCase for URLs.
```

Comment:
I prefer kebab-case for URLs because otherwise you have to worry about making your route checks case-insensitive.